terra-base-jupyter/Dockerfile

```diff
@@ -0,0 +1,198 @@
+# Smallest image with ubuntu jammy, CUDA and NVDIA drivers installed - 80 mb
+FROM --platform=linux/amd64 nvidia/cuda:12.2.0-base-ubuntu22.04
+
+# Use bash as the shell, like the jupyter terminal (just nicer to work with than sh)
+ENV SHELL /usr/bin/bash
+SHELL ["/usr/bin/bash", "-c"]
+
+#######################
+# Environment Variables
+#######################
+ENV DEBIAN_FRONTEND noninteractive
+ENV LC_ALL en_US.UTF-8
+
+# We need node >18 for jupyter to work
+ENV NODE_MAJOR 20
+
+# Set the python version and corresponding conda installer
+ENV PYTHON_VERSION 3.10
+ENV CONDA_INSTALLER https://repo.anaconda.com/miniconda/Miniconda3-py310_23.5.1-0-Linux-x86_64.sh
```

Comment:
Look into this as well: https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community